workflow-templates/im-deploy-az-database.yml

```diff
@@ -0,0 +1,176 @@
+# Workflow Code: BetrayedCod_v1    DO NOT REMOVE
+
+# Prerequisites
+# 1 - Make sure the az secrets have been added to the environment
+# 2 - Make sure the scm restrictions include the prod github runners (previously it had just octopus)
+
+# TODO: Ensure each of the repo-level and env-level secrets used in this workflow have been populated by an admin in your repository.
+# TODO: Set up a deployment board if it does not already exist: https://github.com/im-practices/git-er-done/blob/main/actions/deployment-board.md
+
+name: Deploy Database to Azure
+on:
+  push:
+  workflow_dispatch:
+    inputs:
+      branch-tag-sha:
+        description: The branch, tag or sha of the terraform that should be deployed.  For Production, only tags reachable by the default branch will be accepted.
+        required: true
+      environment:
+        # TODO: Update this description with the correct environments
+        description: 'The environment to deploy to - dev|qa|stage|prod|demo|uat'
+        required: true
+env:
+  GITHUB_REF: ${{ github.event.inputs.branch-tag-sha  }} # This is the branch/tag/sha that we'll be deploying
```

Comment:
If you decide to change this to just tags, the name of this env should be updated to 'RELEASE_TAG' like it is in the other workflows and then all of the places that now reference the GITHUB_TAG would need to be updated.